### PR TITLE
Add to custom_sell() documentation

### PR DESCRIPTION
**Pull request by TheJoeSchr**
- Flesh out infos about return type 
- give quick example at beginning to get reader in right mindset what this does and why it's useful

### CODE REVIEW

File: docs/strategy-advanced.md [hunk 1]
```diff
@@ -44,7 +44,11 @@ class AwesomeStrategy(IStrategy):
 
 ## Custom sell signal
 
-It is possible to define custom sell signals. This is very useful when we need to customize sell conditions for each individual trade, or if you need the trade profit to take the sell decision.
+It is possible to define custom sell signals, indicating that specified position should be sold. This is very useful when we need to customize sell conditions for each individual trade, or if you need the trade profit to take the sell decision. 
+
+For example you could implement a stoploss relative to candle when trade was opened, or a custom 1:2 risk-reward ROI.
```
Comment (xmatthias) [inline docs/strategy-advanced.md]: we explicitly removed the mention of stoploss from here, as using custom-sell-signals for that is a inferior method to using custom_stoploss - which also allows you to keep the stoploss on exchange.

Comment (TheJoeSchr) [reply docs/strategy-advanced.md]: are you changing this right now or should I?

Comment (TheJoeSchr) [reply docs/strategy-advanced.md]: did it and included your explanation

Comment (TheJoeSchr) [reply docs/strategy-advanced.md]: feel free to add additional changes directly, if you think it's still not up to par

File: docs/strategy-advanced.md [hunk 2]
```diff
@@ -44,7 +44,13 @@ class AwesomeStrategy(IStrategy):
...
 ## Custom sell signal
 
-It is possible to define custom sell signals. This is very useful when we need to customize sell conditions for each individual trade, or if you need the trade profit to take the sell decision.
+It is possible to define custom sell signals, indicating that specified position should be sold. This is very useful when we need to customize sell conditions for each individual trade, or if you need the trade profit to take the sell decision. 
+
+For example you could implement a 1:2 risk-reward ROI with `custom_sell()`.
+
+You should abstain from using custom_sell() signals in place of stoplosses though. It is a inferior method to using `custom_stoploss()` in this regard - which also allows you to keep the stoploss on exchange.
+
+!!! Note Returning a `string` or `True` from this method is equal to setting sell signal on a candle at specified time. This method is not called when sell signal is set. `string` max length is 64 characters. Exceeding this limit will raise `OperationalException`.
```
Comment (xmatthias) [inline docs/strategy-advanced.md]: This line has 2 problems.

**Syntax**: after `!!! Note` - you need either a title (in quotes), or a new-line (the next line must be indented by 4 spaces).
**Content**: Exceeding 64 characters will NOT raise an exception (that was the initial implementation, but was not used), but will cut the string after 64 characters.

**Review by xmatthias — APPROVED** [review]
LGTM :+1:

**Pull request merged**